packages/babel-plugin-proposal-enhanced-classes/README.md

```diff
@@ -0,0 +1,19 @@
+# @babel/plugin-proposal-enhanced-classes
```

Comment:
it's not obvious for me how this is supposed to be used (by looking into README), could you add some simple example here?

Reply:
I can't :stuck_out_tongue: (https://github.com/babel/babel/pull/8108)

I will open a new PR containing the docs when this PR is more ready